Sources/QuickSpecBase/QuickSpecBase.m

```diff
@@ -0,0 +1,55 @@
+#import "QuickSpecBase.h"
+
+#pragma mark - _SelectorWrapper
+
+@interface _SelectorWrapper ()
```

Comment:
Let's use a prefix identifier (eg - `_QCKSelectorWrapper` or `_QuickSelectorWrapper`) since it's technically a public class.